src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,185 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String IOOB_MESSAGE = "IndexOutOfBoundsException exception!";
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    public void unlink(int index) {
```

Comment:
```suggestion
    private void unlink(int index) {
```

це допоміжний метод, тому давай зробимо його приватним. І пам'ятай, привтні методи мають знаходитися під публічними 
те саме для інших схожих методів